scss/components/_buttons.scss

```diff
@@ -0,0 +1,19 @@
+.civicase__add-btn {
+  font-weight: $font-weight-heading;
+  line-height: 18px;
+  padding: 8px 18px;
+
+  &::before {
+    background: $crm-white;
+    border-radius: 100%;
+    color: $brand-primary;
+    content: '+';
```

Comment:
why are we using `+` instead of the font awesome or material icon equivalent?

Reply:
Couldn't find the icon in the material icon and font awesome library. If you can direct me to the one I can use it. However I always prefer doing icons with the css. They are cleaner, easily customizable and fast (performant).